tests/phpunit/tests/customize/manager.php

```diff
@@ -3136,7 +3136,7 @@ public function test_remove_frameless_preview_messenger_channel() {
 		ob_start();
 		$manager->remove_frameless_preview_messenger_channel();
 		$output = ob_get_clean();
-		$this->assertStringContainsString( '<script>', $output );
+		$this->assertStringContainsString( '<script', $output );
```

Comment:
would be nice to know what this assertion is supposed to be testing. what's the point here? confirm that we have a `SCRIPT` element in the output?

```php
$processor = new WP_HTML_Tag_Processor( $output );
$this->assertTrue( $processor->next_tag( 'script' ), 'Failed to find expected SCRIPT element in output.' );
```

😉

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/5301/commits/2b764e6edfa29166cb83e5f581ff30cc37974f11 as part of https://github.com/WordPress/wordpress-develop/pull/5301